packages/transaction-controller/src/types.ts

```diff
@@ -574,6 +574,14 @@ export interface DappSuggestedGasFees {
   maxPriorityFeePerGas?: string;
 }
 
+/**
+ * Gas values set by the user.
+ */
+export interface AdvancedGasFees {
```

Comment:
Shall we make this more generic and call it something like `SavedGasFees`?

Reply:
I didn't want to rename it since it's named as `AdvancedGasFees` in the `PrefController` but it all make sense as `SavedGasFees`, done!